spec/requests/v0/disability_compensation_in_progress_forms_controller_spec.rb

```diff
@@ -164,6 +167,32 @@
           end.to change(InProgressForm, :count).by(1)
           expect(response).to have_http_status(:ok)
         end
+
+        it 'adds 0781 metadata if flippers enabled' do
+          Flipper.enable(:disability_compensation_sync_modern0781_flow_metadata)
```

Comment:
For this one and the `Flipper.disable` below, can you mock them instead? See [example here](https://depo-platform-documentation.scrollhelp.site/developer-docs/feature-toggles-guide#FeatureTogglesGuide-Backendexample). Since tests run in parallel in CI, directly enabling/disabling can cause failures.

Reply:
oh yea, for sure.